### DISCUSSION TITLE
Github Achivements

### DISCUSSION THREAD

**Discussion by akileus0000** [community/community#148107] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

Some badges appeared on my github. what're these for?
![image](https://github.com/user-attachments/assets/cc47482a-90a2-45fa-bc3f-3ea828c059e9)

**Comment by bitmanvs** [comment, marked as answer]
Hi @akileus0000,

GitHub Achievements Badges are visual rewards displayed on your GitHub profile to recognize your contributions, activity, and engagement with the GitHub platform.

You can find the reason why you had such badges in detail on your github profile by checking this link:
https://github.com/drknzz/GitHub-Achievements

**Reply by akileus0000** [reply]
Hi @bitmanvs 

I've just checked the link and identified why they appeared on my profile.
Thank you!

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @akileus0000!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Profile category. I've gone ahead and moved it for you. Good luck!
